index.html

```diff
@@ -10,229 +10,233 @@
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" crossorigin="anonymous" rel="stylesheet">
 </head>
 <body>
-    <header>
-        <nav class="flexnav">
-            <a href="index.html"><h3>Welcome</h3></a>
-            <a href="" class="menu">≡</a>
-        </nav>
-    </header>
-
-
+    
     <div class="main">
-
-
-    <section class="landing-page">
-        <div class="hi">
-            <h1>Hello there, I'm Joaquin</h1>
-            <h2 class="enphasis">I'm a software developer</h2>
-        </div>
-
-        <div class="paragraph">
-            <p class="text">
-                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don't hestiate to contact me.
-            </p>
-        </div>
-
-        <ul class="network">
-            <li class="social">
-                <a target="_blank" href="https://github.com/Kakalanp" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/github.png" alt="g"/></a>
-            </li>
-            <li class="social">
-                <a target="_blank" href="https://www.linkedin.com/in/joaquin-garrido-lecca-zanetti-623583204/" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2--v1.png" alt="li"/></a>
-            </li>
-            <li class="social">
-                <a target="_blank" href="https://www.twitter.com/LeccaJoaquin"  rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/twitter--v1.png" alt="t"/></a>
-            </li>
-        </ul>
-    </section>
-
-
-    <section class="works">
-        <section class="title">
-            <h2>My recent works</h2>
-            <img src="/Images/works-horizontal-line.svg" alt="line">
-        </section>
-        
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
+        <header>
+                <nav class="flexnav">
+                    <a href="index.html" class="welcome"><h3>Welcome</h3></a>
+                    <a href="" class="menu">≡</a>
+
+                    <a class="desktop" href="#works-title">Portfolio</a>
+                    <a class="desktop" href="#about">About</a>
+                    <a class="desktop" href="#form">Contact</a>
```

Comment:
- [ ] Your header nav links have no spacing from the top and the left and right spacing are too much.
Can we look at that, and try to make it look like the template from Figma?

Your Design | Figma Design |
:---------------:|:-------------------:|
![e2](https://user-images.githubusercontent.com/39523312/149403676-49d2907d-24f5-4851-8d69-13ec095eed3d.png) |![e1](https://user-images.githubusercontent.com/39523312/149403652-0088c8ab-22f0-4873-b4f2-daa596c7a1e9.png)

Reply:
I'll add more space on top, but the spacing of those links is intentional design, as I think it looks better.